From aa40052f2d0592470c885fad4f63023ad88b04b7 Mon Sep 17 00:00:00 2001
From: Tien Nguyen <tien.nguyen.uh@renesas.com>
Date: Fri, 22 May 2026 08:08:02 +0000
Subject: [PATCH 3/3] media: renesas: rzg2l-cru: adjust AXI burst length

Update the CRU AXI bus master transfer length from 0xf to 0x6.

This uses the validated AXI burst length setting for CRU image data
transfers.

Upstream-Status: Pending

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>
---
 drivers/media/platform/renesas/rzg2l-cru/rzg2l-cru-regs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/renesas/rzg2l-cru/rzg2l-cru-regs.h b/drivers/media/platform/renesas/rzg2l-cru/rzg2l-cru-regs.h
index a161ea8c5061..e6f040eafa29 100644
--- a/drivers/media/platform/renesas/rzg2l-cru/rzg2l-cru-regs.h
+++ b/drivers/media/platform/renesas/rzg2l-cru/rzg2l-cru-regs.h
@@ -38,7 +38,7 @@
 #define AMnMBS_MBSTS			0x7
 
 #define AMnAXIATTR_AXILEN_MASK		GENMASK(3, 0)
-#define AMnAXIATTR_AXILEN		(0xf)
+#define AMnAXIATTR_AXILEN		(0x6)
 
 /* AXI Master FIFO Pointer Register for CRU Image Data */
 #define AMnFIFOPNTR_FIFOWPNTR		GENMASK(7, 0)
-- 
2.43.0

